commit db07dbf5a7e19806b1629bd4125e6643978c6f9f
Author: Dave Anderson <anderson@redhat.com>
Date:   Thu Feb 19 16:16:33 2015 -0500

    Prepare for the future increment of Linux 3.x to 4.x.
    (anderson@redhat.com)

diff --git a/kernel.c b/kernel.c
index cf858c2..a5e0c64 100644
--- a/kernel.c
+++ b/kernel.c
@@ -1211,7 +1211,8 @@ verify_namelist()
 	sprintf(buffer3, "(unknown)");
         while (fgets(buffer, BUFSIZE-1, pipe)) {
 		if (!strstr(buffer, "Linux version 2.") &&
-		    !strstr(buffer, "Linux version 3."))
+		    !strstr(buffer, "Linux version 3.") &&
+		    !strstr(buffer, "Linux version 4."))
 			continue;
 
                 if (strstr(buffer, kt->proc_version)) {
@@ -4909,7 +4910,8 @@ debug_kernel_version(char *namelist)
 	argc = 0;
         while (fgets(buf, BUFSIZE-1, pipe)) {
                 if (!strstr(buf, "Linux version 2.") &&
-		    !strstr(buf, "Linux version 3."))
+		    !strstr(buf, "Linux version 3.") &&
+		    !strstr(buf, "Linux version 4."))
                         continue;
 
 		argc = parse_line(buf, arglist); 

